app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Vaccine is not")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine is outdated")
+        elif not visitor.get("wearing_a_mask", False):
```

Comment:
Using `elif` after a `raise` statement is unnecessary because the `raise` statement will exit the function. You can replace `elif` with `if` to improve code clarity, as suggested in the checklist.